stdlib/LinearAlgebra/src/symmetriceigen.jl

```diff
@@ -184,6 +184,65 @@ function eigen!(A::AbstractMatrix, C::Cholesky; sortby::Union{Function,Nothing}=
     GeneralizedEigen(sorteig!(vals, vecs, sortby)...)
 end
 
+# Bunch-Kaufmann (LDLT) based solution for generalized eigenvalues and eigenvectors
+function eigen(A::StridedMatrix{T}, B::BunchKaufman{T,<:AbstractMatrix}; sortby::Union{Function,Nothing}=nothing) where {T<:BlasFloat}
+    eigen!(copy(A), copy(B); sortby)
```

Comment:
Does this call modify the second argument? It doesn't seem so, but we should check. Like, remove the copy here and then see if `B` is the same after the whole `eigen` computation.

Reply:
Unfortunately, the line `M, TD, p = getproperties!(B)` modifies `B`.